scripts/lib/CIME/case_st_archive.py

```diff
@@ -180,6 +180,7 @@ def _archive_history_files(case, archive, archive_entry,
     if not os.path.exists(archive_histdir):
         os.makedirs(archive_histdir)
         logger.debug("created directory {}".format(archive_histdir))
+    # the compname is drv but the files are named cpl
```

Comment:
Could you please give an example of what files are named cpl currently so that we can have this documented.